lms/pshell.py

```diff
@@ -0,0 +1,25 @@
+from contextlib import suppress
+from transaction.interfaces import NoTransaction
+
+from lms import models
+
+
+def setup(env):
+    request = env["request"]
+
+    request.tm.begin()
+
+    env["tm"] = request.tm
+    request.tm.__doc__ = "Active transaction manager (a transaction is already begun)."
```

Comment:
Writing `__doc__` is the only way to customize the help string banner that pshell prints out at startup